### ISSUE TITLE
Multiline definition

### ISSUE THREAD

**Issue by benzen** [elm-lang/elm-repl#97]
It's weird to have to write one way say:
`````elm
type alias Cell = 
  { value: Int
  , x: Int
  , y: Int
  , animation: Animation
  }
`````
for the compiler.

And another way for the repl:

````elm
type alias Cell = 
  { value: Int
  , x: Int
  , y: Int
  , animation: Animation
  }
````

I would like to just be able to copy past a func in the repl to play with it.
Loading a full file is great, except for the case where i have multiple error in my source, and i want to play with the a function or two in my module before deciding what to do with the errors.

Does this make sense ?

**Comment by jvoigtlaender** [comment]
No, in the way written above, your question does not make much sense. Given that the two pieces of code you show are exactly identical. :smile:

**Comment by benzen** [comment]
My bad. I ve updated the second code block

**Comment by hipertracker** [comment]
Agree. I'd love to have also auto completion and the multi line commands history. See: ipython
And last, but not least, Elm repl still does not recognize type annotations :(
